sdks/python/run_pylint.sh

```diff
@@ -39,14 +39,19 @@ EXCLUDED_GENERATED_FILES=(
 
 FILES_TO_IGNORE=""
 for file in "${EXCLUDED_GENERATED_FILES[@]}"; do
-  if [[ $FILES_TO_IGNORE ]]; then
+  if test -n "$FILES_TO_IGNORE"; then
     FILES_TO_IGNORE="$FILES_TO_IGNORE, "
   fi
   FILES_TO_IGNORE="$FILES_TO_IGNORE$(basename $file)"
 done
 echo "Skipping lint for generated files: $FILES_TO_IGNORE"
 
+if test $# -gt 0
```

Comment:
`pylint` on the whole sdk may take some time. This option helps to run it on the modules you are working on